packages/lib/pcd-types/src/pcd.ts

```diff
@@ -256,32 +256,33 @@ export enum ArgumentTypeName {
   Object = "Object",
   StringArray = "StringArray",
   PCD = "PCD",
-  Record = "Record",
+  RecordContainer = "RecordContainer",
```

Comment:
Compatibility note: I believe this string appears in the serialized form of args, which means this is a breaking change for GPC args in URL requests.

That's fine overall.  I'm intentionally not making any guarantees about compatibility over time for GPCs yet.  But we should be a bit careful about the timing of when this merge happens with respect to active hackers at the hackathon.  Once this merges, a push to prod will have to paired with an NPM package release (and all hackers adopting the new NPM packages) to avoid breaking their apps.